deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +204,67 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payloadEventsString: __ENV.DATA_BTM_SQS_PAYLOAD_EVENTS,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadEventsArray = JSON.parse(eventData.payloadEventsString)
+
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
 const sqs = new SQSClient(awsConfig)
 
-export function sendEvent (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
-  sqs.sendMessage(env.sqs_queue, messageBody)
+export function sendEventType1 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[0])
+  sendSQSMessage(messageBody)
+}
+
+export function sendEventType2 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[1])
+  sendSQSMessage(messageBody)
+}
+
+export function sendEventType3 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[2])
+  sendSQSMessage(messageBody)
+}
+
+export function sendEventType4 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[3])
+  sendSQSMessage(messageBody)
+}
+
+export function sendEventType5 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[4])
+  sendSQSMessage(messageBody)
+}
+
+export function sendEventType6 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[5])
+  sendSQSMessage(messageBody)
+}
+
+export function sendEventType7 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[6])
+  sendSQSMessage(messageBody)
 }
 
-export function sendEventDebug (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
+export function sendSQSMessage (messageBody: string): void {
+  let randomTimestamp: number = 0
+  let timestampFormatted: string = ''
+  randomTimestamp = randomIntBetween(payloadTimestampMin, payloadTimestampMax)
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
```

Comment:
```suggestion
  const randomTimestamp = randomIntBetween(payloadTimestampMin, payloadTimestampMax)
  const timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
```
These can just be `const` declarations

Reply:
https://eslint.org/docs/latest/rules/prefer-const